Content.Shared/GameObjects/Components/SharedExtinguisherComponent.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components
+{
+    public class SharedExtinguisherComponent : Component
+    {
+        public sealed override string Name => "RoguePointingArrow";
```

Comment:
🤔

Reply:
Renamed the shared class to "SharedGasSprayerComponent" and changed the name to "GasSprayer